src/java/picard/sam/MergeSamFiles.java

```diff
@@ -83,6 +90,14 @@
     @Option(doc = "Comment(s) to include in the merged output file's header.", optional = true, shortName = "CO")
     public List<String> COMMENT = new ArrayList<String>();
 
+    @Option(shortName = "INTERVALS", doc = "An interval list file that contains the locations of the positions to merge. "+
+            "Assume bam are sorted and indexed. "+
+            "Reads should not overlap two intervals. "+
```

Comment:
Sorry, my comments are confusing.

I mean to say that the comment "Reads should not overlap two intervals." is unneeded, but that instead there should be a comment warning users from merging the resulting sequence files since they may contain the same read in both files. 

I do not mean that you should add any further checking.